### DISCUSSION TITLE
koodo reader could not be used with Mac mini

### DISCUSSION THREAD

**Discussion by warcowcn** [community/community#118268] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hello, I prefer to use Koodo reader, since it can read books. However, I installed it on the Mac mini, and the machine refuses to use it.

**Comment by LiteBrite82** [comment]
Hi @warcowcn ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/zigbee2mqtt). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
